### ISSUE TITLE
[SUGGESTION] Hide on key hold setting

### ISSUE THREAD

**Issue by srnyx** [maruohon/minihud#182]
A setting that lets you set a key that when held will hide the info toggles.
This would let me hide them when I hold down `TAB` so that they don't clash with anything that is on tab.

Or just a setting that will hide the info toggles when tab is open.

**Comment by MinecraftSeaTurtle** [comment]
This is already possible, though it requires some setting up. Go to the 'generic' section, search for 'toggle' to get to the 'toggleKey' hotkey. Set the hotkey to 'tab' then left click on the small box to the right and turn off 'cancel further processing'. I also recommend turning on 'allow extra keys' and turning off 'exclusive'.

**Comment by srnyx** [comment]
> This is already possible, though it requires some setting up. Go to the 'generic' section, search for 'toggle' to get to the 'toggleKey' hotkey. Set the hotkey to 'tab' then left click on the small box to the right and turn off 'cancel further processing'. I also recommend turning on 'allow extra keys' and turning off 'exclusive'.

Thank you! There were a few more things I had to change, here's what mine looks like for anyone else that wants this:
![image](https://user-images.githubusercontent.com/25808801/144946532-72f6a5a3-f1e5-4031-8341-7c54d23b4500.png)

**Comment by MinecraftSeaTurtle** [comment]
I'm glad I could help, even though I somehow neglected to mention the most important part: changing the 'activate on' to both.

**Comment by srnyx** [comment]
@MinecraftSeaTurtle Discovered an issue with your method. If I'm holding down TAB and then open chat (T), it closes TAB but doesn't unhide the info toggles. Then when I hold down TAB again (when not in chat of course), it reverses (info toggles show when TAB is shown and hide when TAB is hidden).

**Comment by maruohon** [comment]
Yeah this is not an ideal solution for this use case, as you can sometimes miss the release event. So I'll probably add something more suitable for this. But what if you set Context to `Any`? Maybe it would fix at least some of those cases of missing the key release event for now?

**Comment by srnyx** [comment]
> But what if you set Context to `Any`? Maybe it would fix at least some of those cases of missing the key release event for now?

Yeah, it fixes it. But will most likely bring up some other issue, but probably wouldn't happen as much.
